### DISCUSSION TITLE
minor version?

### DISCUSSION THREAD

**Discussion by maticar92** [MaartenGr/BERTopic#1939] [General]
Hi @MaartenGr! First of all thank you so much for creating and maintaining this amazing library. I have been using the merge models functions in v16 extensively. 

Can you share your process to release a minor version with the fixes since v0.16.0 came out? I am limited to installing from pypi and not GitHub and was wondering when the fixes since the v16 would be available or if the next planned release is directly version v0.17

**Comment by MaartenGr** [comment]
Thank you for reaching out! All pull requests that I wanted for a v0.16.1 release has been merged, so I will be working the upcoming couple of days of putting out a new release. I still need to check a couple of things but will mostly make sure the changelog and release properly give credit to those who worked hard on some of these fixes. If it isn't released in a week, feel free to ping me as I'm juggling many things at the moment and might redirect my attention.

**Reply by MaartenGr** [reply]
@maticar92 Done! Took a while to properly credit everyone but I think I didn't miss anyone. The v0.16.1 release is live and can be installed with:

```python
pip install --upgrade bertopic
```

**Comment by maticar92** [comment]
thank you so much I really appreciate it!
